posts/inside-rust/2023-09-19-crates-io-usage-policy-rfc.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: crates.io Policy Update RFC
+author: Tobias Bieniek
+team: the crates.io team <https://www.rust-lang.org/governance/teams/crates-io>
+---
+
+Around the end of July the crates.io team opened an [RFC](https://github.com/rust-lang/rfcs/pull/3463) to update the current crates.io usage policies. This has caused a lot of controversial discussion and most of the suggested improvements have since been integrated into the proposal.
+
+At the last team meeting the crates.io team decided to move the RFC forward and start the **final comment period** process.
+
+We have been made aware by a couple of community members though that the RFC might not have been visible enough in the Rust community. We hope that this blog post changes that.
+
+We invite you all to review the RFC and let us know if there are still any major concerns with these proposed policies.
+
+Here is a quick **TL;DR**:
+
+- The current policies are quite vague on a couple of topics. The new policies are more explicit.
+- Name squatting is allowed, but only to a certain degree and if you have a good reason for it.
+- The crates.io team will try to contact crate owners before taking any actions.
```

Comment:
This could imply that the crates.io team won't _always_ try to contact crate owners. How about:

```suggestion
- The crates.io team will always attempt to contact crate owners before taking any actions.
```